app/controllers/candidate_interface/referees_controller.rb

```diff
@@ -118,5 +140,12 @@ def referee_params
     def redirect_to_review_referees_if_amendable
       redirect_to candidate_interface_review_referees_path if current_application.amendable?
     end
+
+    def send_slack_message
+      message = "A referee has cancelled their request for a reference from #{@referee.name}."
```

Comment:
We shouldn't use full names/other PII in Slack messages, perhaps this?

```suggestion
      message = "Candidate #{candidate_first_name} has cancelled one of their references"
```